{"timestamp":"2025-10-21T10:03:10.843291Z","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager","filename":"manager.py","lineno":179}
{"timestamp":"2025-10-21T10:03:10.844250Z","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/main.py","logger":"airflow.models.dagbag.DagBag","filename":"dagbag.py","lineno":593}
{"timestamp":"2025-10-21T10:03:11.697242Z","level":"info","event":"Task instance is in running state","logger":"task.stdout"}
{"timestamp":"2025-10-21T10:03:11.697740Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","logger":"task.stdout"}
{"timestamp":"2025-10-21T10:03:11.698396Z","level":"info","event":"Current task name:send_email","logger":"task.stdout"}
{"timestamp":"2025-10-21T10:03:11.698903Z","level":"info","event":"Dag name:Airflow_Lab2","logger":"task.stdout"}
{"timestamp":"2025-10-21T10:03:11.708235Z","level":"error","event":"Task failed with exception","logger":"task","filename":"task_runner.py","lineno":972,"error_detail":[{"exc_type":"AirflowException","exc_value":"SMTP connection is not found.","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":920,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1307,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":416,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/smtp/operators/smtp.py","lineno":89,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/smtp/hooks/smtp.py","lineno":79,"name":"__enter__"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/smtp/hooks/smtp.py","lineno":97,"name":"get_conn"}],"is_group":false,"exceptions":[]},{"exc_type":"AirflowNotFoundException","exc_value":"The conn_id `smtp_default` isn't defined","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/smtp/hooks/smtp.py","lineno":95,"name":"get_conn"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/hook.py","lineno":61,"name":"get_connection"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/definitions/connection.py","lineno":228,"name":"get"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/definitions/connection.py","lineno":218,"name":"_handle_connection_error"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-10-21T10:03:11.709473Z","level":"info","event":"Task instance in failure state","logger":"task.stdout"}
{"timestamp":"2025-10-21T10:03:11.710014Z","level":"info","event":"Task start","logger":"task.stdout"}
{"timestamp":"2025-10-21T10:03:11.710332Z","level":"info","event":"Task:<Task(EmailOperator): send_email>","logger":"task.stdout"}
{"timestamp":"2025-10-21T10:03:11.710659Z","level":"info","event":"Failure caused by SMTP connection is not found.","logger":"task.stdout"}
